expyfun/stimuli/_stimuli.py

```diff
@@ -131,3 +131,57 @@ def play_sound(sound, fs=None, norm=True, wait=False):
     if hasattr(snd, 'delete'):  # for backward compatibility
         Timer(del_wait, snd.delete).start()
     return snd
+
+
+def add_pad(sounds, alignment='start'):
+    """Add sounds of different lengths and channel counts together
+
+    Parameters
+    ----------
+    sounds : list
+        The sounds to add together. There is a maximum of two channels.
+    alignment : str
+        How to align the sounds. Either by ``'start'`` (add zeros at end),
+        ``'center'`` (add zeros on both sides), or ``'end'`` (add zeros at
+        start).
+
+    Returns
+    -------
+    y : float
+        The summed sounds. The number of channels will be equal to the maximum
+        number of channels (by appending a copy), and the length will be equal
+        to the maximum length (by appending zeros).
+
+    Notes
+    -----
+        Even if the original sounds were all 0- or 1-dimensional, the output
+        will be 2-dimensional (channels, samples).
+    """
+    if alignment not in ['start', 'center', 'end']:
+        raise(ValueError("alignment must be either 'start', 'center', "
+                         "or 'end'"))
+    x = [np.atleast_2d(y) for y in sounds]
+    if not np.all([y.ndim == 2 for y in x]):
```

Comment:
no need for `[` enclosing here

better `all(y.ndim == 2 for y in x)` as it's a generator expression that will short-circuit